src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Users/UserMapper.cs

```diff
@@ -12,6 +12,9 @@ public UserMapper()
       .ForCtorParam(nameof(Managers.Registry.Contract.Registrants.UserProfile.DateOfBirth), opts => opts.MapFrom(s => s.DateOfBirth))
       .ForCtorParam(nameof(Managers.Registry.Contract.Registrants.UserProfile.Email), opts => opts.MapFrom(s => s.Email))
       .ForCtorParam(nameof(Managers.Registry.Contract.Registrants.UserProfile.Phone), opts => opts.MapFrom(s => s.Phone))
+      .ForCtorParam(nameof(Managers.Registry.Contract.Registrants.UserProfile.MiddleName), opts => opts.MapFrom(s => s.MiddleName))
```

Comment:
Not needed here, essentially you can treat these fields as the Address fields

Reply:
@ytqsl, please see my new changes to `UserMapper()`